pkg/util/goroutinemap/goroutinemap.go

```diff
@@ -22,19 +22,18 @@ with the same name exists.
 package goroutinemap
 
 import (
-	"fmt"
-	"runtime"
 	"sync"
 	"time"
 
 	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/util/goroutinemap/common"
 	k8sRuntime "k8s.io/kubernetes/pkg/util/runtime"
 )
 
 const (
 	// initialDurationBeforeRetry is the amount of time after an error occurs
-	// that GoRoutineMap will refuse to allow another operation to start with
```

Comment:
@saad-ali saad-ali Maybe do the renaming in a separate PR. IT confuses the changes you make here. And really should not go in a 1.3 fix.